Source/Shaders/ShadowVolumeVS.glsl

```diff
@@ -17,7 +20,11 @@ vec4 depthClampFarPlane(vec4 vertexInClipCoordinates)
 void main()
 {
     v_color = color;
-    
+
     vec4 position = czm_computePosition();
+    float delta = min(u_globeMinimumAltitude, czm_geometricToleranceOverDistance * length(position.xyz));
+
+    //extrudeDirection is zero for the top layer
+    position = position + vec4(extrudeDirection.xyz * delta, 0);
```

Comment:
Use `0.0`, mixing integers and floats didn't use to compile on some WebGL implementations.